sdk/src/main/java/io/opentelemetry/sdk/metrics/aggregator/Aggregator.java

```diff
@@ -57,4 +57,7 @@
    * @param value the new {@code double} value to be added.
    */
   void recordDouble(double value);
+
+  /** Whether there have been any recordings since this aggregator has been reset. */
+  boolean hasRecordings();
```

Comment:
No particular preference but wondered if `isEmpty` is more idiomatic

Reply:
In this case, I don't know that aggregators necessarily have a universal concept of "emptiness", so I went with something very semantically clear and domain-specific.